libraries/app/include/scorum/app/database_api.hpp

```diff
@@ -28,9 +28,12 @@
 namespace scorum {
 namespace app {
 
+using std::set;
```

Comment:
What the difference using std::set; or using std;? I think we agreed to remove all variants of using std?